page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/SetupStepDefs.java

```diff
@@ -15,29 +13,64 @@
 
 public class SetupStepDefs {
 
-    @Before(order = 0)
-    public void preSetUp(Scenario scenario) {
+    private static final ThreadLocal<Boolean> ispreSetUp = ThreadLocal.withInitial(() -> false);
+    private static final ThreadLocal<Boolean> isSetUp = ThreadLocal.withInitial(() -> false);
+    private static final ThreadLocal<Boolean> isTearDown = ThreadLocal.withInitial(() -> false);
+
+    private SetupStepDefs() {
+        throw new AssertionError("Instantiating utility class.");
+    }
+
+    public static void preSetUp(Scenario scenario) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "scenario". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)